bpython/debugger.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python2.7
+# -*- coding: utf-8 -*-
+# vim:ts=4:sw=4:softtabstop=4:smarttab:expandtab
+
+"""
+Debugger factory. Set PYTHON_DEBUGGER to a module path that has a post_mortem
+function in it. Defaults to pdb. This allows alternate debuggers to be used,
+such as pycopia.debugger. :)
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+import os
+import sys
+
+
+post_mortem = None
+
+def _get_debugger():
```

Comment:
It will raise an ImportError so that the user (who presumably knows what he/she is doing if they are going to set PYTHON_DEBUGGER) can fix it. 

Only "post_mortem" is looked for.  "pm" is a local variable. 

Sometimes it's best to just let errors pass up, so the user can get a full stack trace and know where to fix it.